src/worker.cc

```diff
@@ -1079,15 +922,14 @@ void from_json(rapidjson::Document& doc, Options& options) {
 
 namespace valhalla {
 
-valhalla_exception_t::valhalla_exception_t(unsigned code, const boost::optional<std::string>& extra)
-    : std::runtime_error(""), code(code), extra(extra) {
-  auto code_iter = error_codes.find(code);
-  message = (code_iter == error_codes.cend() ? "" : code_iter->second);
-  message += (extra ? ":" + *extra : "");
-  auto http_code_iter = ERROR_TO_STATUS.find(code);
-  http_code = (http_code_iter == ERROR_TO_STATUS.cend() ? 0 : http_code_iter->second);
-  auto http_message_iter = HTTP_STATUS_CODES.find(http_code);
-  http_message = (http_message_iter == HTTP_STATUS_CODES.cend() ? "" : http_message_iter->second);
+valhalla_exception_t::valhalla_exception_t(unsigned code, const std::string& extra)
+    : std::runtime_error("") {
+  auto code_itr = error_codes.find(code);
+  if (code_itr != error_codes.cend()) {
+    *this = code_itr->second;
```

Comment:
so now if you specify just the valhalla error code we look up a predefined exception from above and return that.